tempel.el

```diff
@@ -108,7 +108,7 @@ must return a list of templates which apply to the buffer or context."
 (defvar tempel--file-templates nil
   "Templates loaded from the `tempel-file'.")
 
-(defvar tempel--file-modified nil
+(defvar tempel--paths-modified nil
```

Comment:
path-modified

Reply:
I'll change the `paths` to `path` to match with `tempel-path`.